AtlasTutorial/Summer2019/README.md

```diff
@@ -0,0 +1,9 @@
+# ATLAS tutorial
+
+## Slides
+
+The slides can be found in this folder as PDF.
+
+## Hands-on
+
+The hands-on is carried out using the Jupyter notebooks, which can be found [here](https://github.com/stwunsch/root_dataframe_tutorial/tree/atlas-tutorial-summer2019) or in this folder. Since all participants have a CERN account, the notebooks are run on [SWAN](https://swan.web.cern.ch) but the notebooks are also runnable using [Binder](https://mybinder.org).
```

Comment:
Do you still want to reference your repo?

Reply:
I would keep it in, why not? I've done special branches for each tutorial, should be fine.

Reply:
Perhaps I misunderstood, but wasn't your goal to have everything available in the ROOT repo and centralized? Why keeping two repos?